Material.Blazor/Components/TopAppBar/MBTopAppBar.razor

```diff
@@ -3,14 +3,16 @@
 
 
 <header @ref="HeaderElem"
-        @attributes="AttributesToSplat(SplatType.ExcludeEvents)"
-        id="app-bar">
```

Comment:
This is most certainly a bug which is now fixed. We shouldn't hard-code an id in the library